src/providers/types/aptos.d.ts

```diff
@@ -29,5 +29,6 @@ export declare interface AptosProviderInterface extends BloctoProviderInterface
   signTransaction(transaction: any): Promise<SubmitTransactionRequest>;
   signMessage(payload: SignMessagePayload): Promise<SignMessageResponse>;
   disconnect(): Promise<void>;
+  address?: string
```

Comment:
Do we need to expose this? I suppose we want the users to get the address from `provider.publicAccount.address`